styles/build.scss

```diff
@@ -0,0 +1 @@
+@import 'angular-patternfly';
```

Comment:
I think this file needs to be named angular-patternfly.scss for it to be intuitive for consumption. This likely means moving the partials into a sub-directory or adding all the partials here rather than having an _angular-patternfly.scss

Reply:
@TheRealJon recommended the main partial:
"Yes. The build.scss file is for dev/test purposes to build css from the sass (since sass ignores partials by default, you can't compile from a file that starts with _). The _angular-patternfly.scss partial would ship in dist so that it could be consumed. That way consumers don't have to import individual partials. Then build.scss could also just @import 'angular-patternfly';."

Reply:
It doesn't **have** to start with an underscore, but we are doing it that way in patternfly core, so I figured it would be better to keep things consistent. It's also done similarly in other libraries like bootstrap-sass and font-awesome-sass.

Reply:
Just tested renaming to "angular-patternfly.scss" and css generated correctly.   Just want to make sure that "...The angular-patternfly.scss partial would ship in dist so that it could be consumed. That way consumers don't have to import individual partials.."  is still correct.  In other words, app devs can sill consume (@import?)  'dist/angular-patternfly.scss', even though it doesn't begin with an underscore.  -thanks

Reply:
OK, this is fine then.

Reply:
Just as a reference:

[Sass Partials](http://sass-lang.com/guide#topic-4)
[Sass Imports](http://sass-lang.com/guide#topic-5)

Sass does a bit of magic with imports and file names, so those sections definitely cleared some confusion up for me.